terraform/cyhy_private_dns.tf

```diff
@@ -82,9 +82,9 @@ resource "aws_route53_record" "cyhy_vulnscan_A" {
 resource "aws_route53_zone" "cyhy_scanner_zone_reverse" {
   # NOTE:  This assumes that we are using /24 blocks
   name = "${format("%s.%s.%s.in-addr.arpa.",
-    element( split(".", aws_subnet.cyhy_scanner_subnet.cidr_block) ,2),
```

Comment:
Good catch... ?
How about that ") ,2)"

Reply:
Now fixed.  I originally saw the spaces at the beginning but saw no corresponding spaces at the end.